include/dem/grid_motion.h

```diff
@@ -109,14 +111,49 @@ class GridMotion
   move_grid_translational(
     parallel::distributed::Triangulation<dim> &triangulation);
 
+  /**
+   * Carries out translational motion of the triangulation
+   *
+   * @param triangulation Triangulation
+   */
+  void
+  move_grid_due_particles_forces(
+    parallel::distributed::Triangulation<dim> &triangulation);
```

Comment:
The naming choice here is weird. Is this moving the grid, or integrating the equation of motion for the grid? Use complete sentences or wording when establishing function names.

Reply:
like move_grid_due_to_particles_forces is a better name @blaisb ?